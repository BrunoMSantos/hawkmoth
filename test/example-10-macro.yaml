directive-arguments:
- example-10-macro.c
