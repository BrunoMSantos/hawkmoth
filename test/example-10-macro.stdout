
.. c:macro:: FAILURE

   Failure status.


.. c:function:: DIE()

   Terminate immediately with failure status.

   See :c:macro:`FAILURE`.


.. c:function:: ARRAY_SIZE(array)

   Get the number of elements in an array.

   :param array: An array
   :return: Array size


.. c:function:: VARIADIC_MACRO(foo, ...)

   Variadic macros

   :param foo: regular argument
   :param ...: variable argument

